packages/clay-date-picker/src/Helpers.ts

```diff
@@ -17,6 +19,36 @@ export type WeekDays = Array<IDay>;
 
 export type Month = Array<WeekDays>;
 
+export const RANGE_SEPARATOR = ' - ';
+
+export function isDateRangeWithinYears(year: number, years: IYears) {
```

Comment:
This function name doesn't seem to describe what it does. Name says that it tells you whether a date range (presumably with a start and end) falls within `years` (which implies a range). But it looks to me like what it actually does is tell you whether a year falls within a range (of years).

So maybe it should be called something like `isYearWithinYearRange()` (or `isYearWithinYears()`).

Reply:
isYearWithinYears seems appropriate :)